libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/mysql/MySQLNoLessThan5700SchemaAccessor.java

```diff
@@ -856,7 +856,15 @@ public Map<String, DBTablePartition> listTablePartitions(@NonNull String schemaN
         return tableName2Res.entrySet().stream().collect(Collectors.toMap(Entry::getKey, e -> {
             Map<String, List<Map<String, Object>>> partiName2Res = e.getValue().stream()
                     .collect(Collectors.groupingBy(m -> (String) m.get("PARTITION_NAME")));
-            List<Map<String, Object>> group = partiName2Res.keySet().stream().map(partiName -> {
+            List<String> orderedPartiName = new ArrayList<>();
```

Comment:
may use linkedhashmap, for avoid possible performance risk while there exists too much partition for the table

Reply:
I use LinkedhashMap instead~